module/entities/helper.js

```diff
@@ -917,7 +919,7 @@ module.exports = class EntitiesHelper {
    */
 
     static search(
-        entityTypeId, 
```

Comment:
@VISHNUDAS-tunerlabs check all the places where search function is called and replace entityTypeId with entityType

Reply:
search is called here only